### ISSUE TITLE
Question about direction head and loss

### ISSUE THREAD

**Issue by bjzhb666** [TonyXuQAQ/UnofficialLaneExtraction#2]
Thanks for your code. I have a question about the direction head and loss.

I find that the direction head is calculated by https://github.com/TonyXuQAQ/UnofficialLaneExtraction/blob/master/code/cnnmodels/resnet34unet.py#L89-L95 and just use a FFN without any constraints.
I wonder if we need to add some constraints like limiting the output values from -1 to +1? Or some other constraints (e.g. loss) make the sum of the square of sin and cos value equal to 1.

What is the performance of the current version? Is the direction accurate? 

I am not very familiar with tensorflow. Hope you can give me some guidelines.

**Comment by FillNullStudio** [comment]
Thank you for the code. I wanna ask the code mentioned there is pretrained model here https://drive.google.com/uc?id=1nnSxgvBzpXyt6vhir8cCDsbhVIOxy33Z. But I cannot find it alreadt. Could you please share it again?

Thank you so much.
